docs/examples/Reinforcement_Learning_Sensor_Manager.py

```diff
@@ -0,0 +1,1002 @@
+from stonesoup.measures import Euclidean
+siap_generatorA = SIAPMetrics(position_measure=Euclidean((0, 2)),
+                              velocity_measure=Euclidean((1, 3)),
+                              generator_name='Reinforcement',
+                              tracks_key='tracksA',
+                              truths_key='truths')
+
+siap_generatorB = SIAPMetrics(position_measure=Euclidean((0, 2)),
+                              velocity_measure=Euclidean((1, 3)),
+                              generator_name='BruteForceSensorManager',
+                              tracks_key='tracksB',
+                              truths_key='truths')
+
+from stonesoup.dataassociator.tracktotrack import TrackToTruth
+associator = TrackToTruth(association_threshold=30)
+
+from stonesoup.metricgenerator.uncertaintymetric import SumofCovarianceNormsMetric
+uncertainty_generatorA = SumofCovarianceNormsMetric(generator_name='ReinforcementSensorManager',
+                                                    tracks_key='tracksA')
+
+uncertainty_generatorB = SumofCovarianceNormsMetric(generator_name='BruteForceSensorManager',
+                                                    tracks_key='tracksB')
+
+# %%
+# Generate a metrics manager.
+
+from stonesoup.metricgenerator.manager import MultiManager
+
+metric_manager = MultiManager([ospa_generatorA,
+                               ospa_generatorB,
+                               siap_generatorA,
+                               siap_generatorB,
+                               uncertainty_generatorA,
+                               uncertainty_generatorB],
+                               associator=associator)
+
+# %%
+# For each time step, data is added to the metric manager on truths and tracks. The metrics themselves can then be
+# generated from the metric manager.
+
+metric_manager.add_data({'truths': truths, 'tracksA': tracksA, 'tracksB': tracksB})
+
+metrics = metric_manager.generate_metrics()
+
+# %%
+# OSPA metric
+# -----------
+# First we look at the OSPA metric. This is plotted over time for each sensor manager method.
+
+from stonesoup.plotter import MetricPlotter
+
+fig = MetricPlotter()
+fig.plot_metrics(metrics, metric_names=['OSPA distances'])
+
+# %%
+# The :class:`~.BruteForceSensorManager` generally results in a smaller OSPA distance
+# than the observations of the :class:`~.ReinforcementSensorManager`, reflecting the better tracking performance
+# seen in the tracking plots.
+
+# %%
+# SIAP metrics
+# ------------
+# Next we look at SIAP metrics. We are only interested in the positional accuracy (PA) and velocity accuracy (VA).
+# These metrics can be plotted to show how they change over time.
+
+
+fig2 = MetricPlotter()
+fig2.plot_metrics(metrics, metric_names=['SIAP Position Accuracy at times',
+                                         'SIAP Velocity Accuracy at times'])
+
+# %%
+# Similar to the OSPA distances the :class:`~.BruteForceSensorManager`
```

Comment:
These are labelled 'accuracy', but smaller is better isn't it? And ---aaaaarrgh -- you've switched the colours!

Reply:
This is part of the metrics manager class in Stone Soup, so I'm not sure why it switches colours halfway through. When I initialise the metric manager, it's all done in the same order.

Reply:
Raise an issue/report a bug with the `MetricManager`?

Reply:
Or perhaps just force it by swapping the order in the metrics managers until you get a match.

Reply:
This ended up being my own error, the metric generator names must match